README.md

```diff
@@ -42,8 +42,42 @@ $ cmake ..
 $ cmake --build .
 ```
 
-This will build Fizzy as a library and since there is no public API
-(the so called *embedder API* in WebAssembly) yet, this is not very useful.
+This will build Fizzy as a library. [C API] is provided for embedding Fizzy engine in applications.
+
+A small example of using the C API, which loads a wasm binary and executes a function named `main`:
+
+```c
+#include <fizzy/fizzy.h>
+
+bool execute_main(const uint8_t* wasm_binary, size_t wasm_binary_size)
```

Comment:
It would be nice to create matching integration test at some point to make sure this example continue to work.